RELEASE-LEADS.md

```diff
@@ -75,29 +75,19 @@ https://github.com/knative/community/blob/master/peribolos/knative-sandbox.yaml
 [this one](https://github.com/knative/community/pull/209) to grant yourself some
 super power.
 
-### Revert all pins to pin master branches again
+### Create a release Slack channel
 
-Revert all pins in all repositories to pin the **master** branches again, run
-`hack/update-deps.sh --upgrade` and PR the changes.
+Ask someone from the TOC to create a **release-`#`** Slack channel that will be
+used to help manage this release.
 
-You should only need to do this for
-`knative/{serving,eventing-contrib,eventing}` and
-`knative-sandbox/net-{istio,contour,kourier,http01,certmanager}`. However, you
-may want to double check `knative/{pkg,caching,networking}` as well in case the
-previous release leads missed a step during their last rotation.
-
-Example PRs:
+## 14 days prior to the release
 
-- [knative/serving](https://github.com/knative/serving/pull/8579)
-- [knative/eventing](https://github.com/knative/eventing/pull/3546)
-- [knative/eventing-contrib](https://github.com/knative/eventing-contrib/pull/1272)
-- [knative-sandbox/net-istio](https://github.com/knative-sandbox/net-istio/pull/172)
-- [knative-sandbox/net-contour](https://github.com/knative-sandbox/net-contour/pull/154)
-- [knative-sandbox/net-kourier](https://github.com/knative-sandbox/net-kourier/pull/84)
-- [knative-sandbox/net-http01](https://github.com/knative-sandbox/net-http01/pull/42)
-- [knative-sandbox/net-certmanager](https://github.com/knative-sandbox/net-certmanager/pull/39)
+### Update the Knative releasability defaults
 
-## 14 days prior to the release
+Update the defaults in
+[knative-releasability.yaml](https://github.com/knative-sandbox/.github/blob/master/workflow-templates/knative-releasability.yaml#L36-L41)
```

Comment:
Would it be better to use permalink since the code location might change in the master branch?